app/translate_survey.py

```diff
@@ -96,6 +96,10 @@ def translate_survey(survey_json, translations):
                     translate_titles_text(question, question['id'], translations)
                     translate_definitions_text(question, question['id'], translations)
 
+                    if 'answers' not in question:
+                        translate_container(question, question['id'], translations)
```

Comment:
Do we need this? the arguments are identicle to https://github.com/ONSdigital/eq-translations/pull/22/files#diff-a8244b82dd0f2093e88fb9263c12af67R93